packages/components/badge/src/Badge.vue

```diff
@@ -60,7 +60,7 @@ const useCountValue = (
         return false
       }
       if (isNumeric(props.count) && isNumeric(overflowCount.value)) {
-        return props.count > overflowCount.value ? `${overflowCount.value}+` : `${props.count}`
+        return toNumber(props.count) > overflowCount.value ? `${overflowCount.value}+` : `${props.count}`
```

Comment:
这样修复是没有问题的，不过我有个建议：
```suggestion
        const overflowCount = toNumber(overflowCount.value, Number.MAX_VALUE)
        return props.count > overflowCount ? `${overflowCount}+` : `${props.count}`
```

同时移除 if 中的 `&&  isNumeric(overflowCount.value)`，并添加一个 `overflowCount` 为非数字字符串的测试用例。